### ISSUE TITLE
GNOME Tweaks showing inconsistently in search results

### ISSUE THREAD

**Issue by cassidyjames** [system76/pop-shop#16]
As reported at https://github.com/system76/pop-distro/issues/99. Not sure if it's reproducible or if there's anything for us to actually do, but filing against the correct repo here.

**Comment by cassidyjames** [comment]
I think @jackpot51 had a clue:
>I wonder if it is being translated, and is searching the translations.

I am unable to reproduce on my machine in English, but in the user-provided screenshots, they're using a different language and the name of the app is translated.

**Comment by brs17** [comment]
@cassidyjames do you know if this issue is reproducible on elementary's appcenter?

**Comment by cassidyjames** [comment]
@brs17 almost certainly. We haven't touched any of the search code in Pop!_Shop afaik. I can test when I get a chance, but it will require installing an alternate language and trying with that.

**Comment by brs17** [comment]
If you have the time, otherwise I'll give it a shot and try and recreate the issue in Pop!_OS and then elementary

**Comment by brs17** [comment]
So this definitely seems related to translations. I just did a Spanish install for Pop!_OS and I see that gnome-tweak-tool is called `Retoques de GNOME`. `gnome tweak` and `tweak` both do not return the tweak tool. Going to try in elementary next.

**Comment by brs17** [comment]
Well now I have to ask the question...what exactly is the issue here? In a foreign language Gnome Tweaks does not go by that name so I would kind of expect the current functionality. Should all searches also be searching the English translation? What about an application that isn't English by default?

**Comment by cassidyjames** [comment]
@brs17 yeah I think searching the current language makes sense. But my concern is that the package ID and binary name include "tweak", where that doesn't return the app. So it sounds like the actual issue is that PackageKit search doesn't appear to be searching the ID or binary names.

**Comment by hartraft** [comment]
I can't find Gnome Tweaks in Pop Shop and had to install via command line using apt. Has there been any updates on PackageKit?
